src/Link.php

```diff
@@ -53,31 +52,31 @@ public function __construct($relation, string $uri = '', bool $isTemplated = fal
     /**
      * {@inheritDoc}
      */
-    public function getHref()
+    public function getHref(): string
```

Comment:
This class is not `final`: adding return types here will be a major BC break for downstream